docs/atomic-rollbacks.md

```diff
@@ -0,0 +1,174 @@
+---
+nav_order: 6
+---
+
+# Atomic Rollbacks
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+## Automatic rollbacks
+
+See [greenboot](https://github.com/fedora-iot/greenboot/blob/main/README.md) for information on automatic rollbacks and how to integrate
+without your bootloader.
+
+## Manual rollbacks
+
+To manually rollback, an older entry can be selected via GRUB bootloader or in
```

Comment:
ostree isn't tied to grub, so I would say e.g.

> ostree writes bootloader entries that are interpreted by the bootloader.  For bootloaders such as GRUB and syslinux that have an interactive UI, it is possible to select a previous boot entry.  For the Android bootloader...

Reply:
SGTM